src/templates/app/action-page.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import React from 'react'
+import _ from 'lodash'
+import { StaticQuery, graphql } from 'gatsby'
+import { DataDuesPage } from './'
+
+const componentMap = {
+  'data-dues': DataDuesPage
+}
+
+type Props = {
+  slug: $Keys<typeof componentMap>,
+  user: User
+}
+
+const ActionPageTemplate = (props: Props) => {
+  const { slug, user } = props
+
+  return (
+    <StaticQuery
+      query={graphql`
+        query {
+          # TODO: we need to avoid to fetch all the created MarkdownRemark nodes
+          allMarkdownRemark {
```

Comment:
Is it possible to do something like this to query by slug? https://www.gatsbyjs.org/docs/adding-markdown-pages/#create-a-page-template-for-the-markdown-files

Reply:
That's the ideal thing I (probably we) want to do, I think the note on the PR is too confusing, but the problem I face is that remember that this page of action is a private page "client only page" that prevent me to do that ideal solution where we can have only a markdown template page of actions and render from there.

_I will update a bit the description of the Pull Request_

Reply:
@duranmla cool thanks!